notebooks/llm-chatbot/llm-chatbot-generate-api.ipynb

```diff
@@ -0,0 +1,1536 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.            with open("llm_config.py", "w") as f:
</pre><p>encoding="utf-8"</p><p> </p><p>this file contains chinese characters, it is impossible to read it without encoding</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2093/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>done</p>